tests/search/test_models.py

```diff
@@ -84,6 +84,15 @@ def test_get_data_should_return_all_data_by_publisher(self):
         dpq = DataPackageQuery(query_string)
         self.assertEqual(0, len(dpq.get_data()))
 
+    def test_should_return_data_package_containing_query(self):
```

Comment:
There is a missing test to query for some non-existing text, that should return 0 results.